spec.html

```diff
@@ -107,6 +107,30 @@ <h1>Well-Known Intrinsic Objects</h1>
 <emu-clause id="sec-abstract-operations">
   <h1>Abstract Operations</h1>
 
+  <emu-clause id="sec-SetterThatIgnoresPrototypeProperties" type="abstract operation">
+    <h1>
+      SetterThatIgnoresPrototypeProperties (
+        _home_: an Object,
+        _p_: a property key,
+        _v_: an ECMAScript language value,
+      ): either a normal completion containing ~unused~ or a throw completion
+    </h1>
+    <dl class="header">
+    </dl>
+    <emu-alg>
+      1. Let _O_ be ? RequireObjectCoercible(*this* value).
+      1. If _O_ is _home_, then
+        1. NOTE: Throwing here emulates assignment to a non-writable data property on the _home_ object in strict mode code.
+        1. Throw a *TypeError* exception.
+      1. Let _desc_ be ? _O_.[[GetOwnProperty]](_p_).
+      1. If _desc_ is *undefined*, then
+        1. Perform ? CreateDataPropertyOrThrow(_O_, _p_, _v_).
```

Comment:
I figured the unenumerability in the previous iteration was intentional given typical "constructor" and @@toStringTag properties is enumerable and the accessor property itself won’t be either, but I agree this is likely the better behavior due to to being less unique / more similar to OrdinarySet.

Reply:
Yeah it looks like the override mistake does not cause the new property to inherit the enumerability or configurability of the existing property on the prototype. So making it enumerable/writable/configurable is definitely the correct thing to do.